peazip-sources/peach.pas

```diff
@@ -49612,7 +49617,11 @@ procedure TForm_peach.mpeautilsClick(Sender: TObject);
    cl:ansistring;
 begin
 P:=tprocessutf8.Create(nil);
-cl:=stringdelim(escapefilename(executable_path,desk_env)+'res'+DirectorySeparator+'pea'+EXEEXT);
+{$IFDEF DARWIN}
+cl:='open ' + executable_path + '../../../pea.app';
+{$ELSE}
+cl:=stringdelim(escapefilename(executable_path,desk_env)+'pea'+EXEEXT);

```

Comment:
pea binary is in (peazip)/res subfolder

Reply:
In Macos, both peazip.app and pea.app should be in the /Applications directory. Of course, if you don’t want to expose pea.app, you can also put pea.app in Resources or other self-built directories.

Reply:
I understand. In the updated sources published yesterday in the peazip-sources directory I've left unchanged your conditional compilation code for Darwin and changed the code for other platforms.

{$IFDEF DARWIN}
cl:='open ' + executable_path + '../../../pea.app';
{$ELSE}
cl:=stringdelim(escapefilename(resource_path,desk_env)+'pea'+EXEEXT);
{$ENDIF}

If this is OK for Darwin, I'll use the same conditional compilation mechanism for the other instances pea binary is invoked, eg. compose_pea_cl, compose_rfs_cl etc.